{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Rockhopper analysis"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Set path"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "ref = 'ann/mtuTemp/'\n",
    "fastq = '../data/fastq/'\n",
    "result1 = 'Results_cond0.1_6'\n",
    "result2 = 'Results_cond50_6'\n",
    "result3 = 'Results_cond50_0.1'"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Cond_0.1_vs_Cond_6"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Aligning sequencing reads from files:\n",
      "\t4_S4_L001_R1_001.fastq.gz\n",
      "\t4_S4_L001_R2_001.fastq.gz\n",
      "Total reads:            \t3061062\n",
      "Successfully aligned reads:\t2667646\t87%\t(>NC_000962.3 Mycobacterium tuberculosis H37Rv, complete genome)\n",
      "\tAligning (sense) to protein-coding genes:\t1%\n",
      "\tAligning (antisense) to protein-coding genes:\t12%\n",
      "\tAligning (sense) to ribosomal RNAs:\t0%\n",
      "\tAligning (antisense) to ribosomal RNAs:\t0%\n",
      "\tAligning (sense) to transfer RNAs:\t0%\n",
      "\tAligning (antisense) to transfer RNAs:\t0%\n",
      "\tAligning (sense) to miscellaneous RNAs:\t0%\n",
      "\tAligning (antisense) to miscellaneous RNAs:\t84%\n",
      "\tAligning to unannotated regions:        \t2%\n",
      "\n",
      "Aligning sequencing reads from files:\n",
      "\t2_S2_L001_R1_001.fastq.gz\n",
      "\t2_S2_L001_R2_001.fastq.gz\n",
      "Total reads:            \t3124230\n",
      "Successfully aligned reads:\t1876505\t60%\t(>NC_000962.3 Mycobacterium tuberculosis H37Rv, complete genome)\n",
      "\tAligning (sense) to protein-coding genes:\t1%\n",
      "\tAligning (antisense) to protein-coding genes:\t5%\n",
      "\tAligning (sense) to ribosomal RNAs:\t0%\n",
      "\tAligning (antisense) to ribosomal RNAs:\t0%\n",
      "\tAligning (sense) to transfer RNAs:\t0%\n",
      "\tAligning (antisense) to transfer RNAs:\t0%\n",
      "\tAligning (sense) to miscellaneous RNAs:\t0%\n",
      "\tAligning (antisense) to miscellaneous RNAs:\t93%\n",
      "\tAligning to unannotated regions:        \t1%\n",
      "\n",
      "Aligning sequencing reads from files:\n",
      "\t1_S1_L001_R1_001.fastq.gz\n",
      "\t1_S1_L001_R2_001.fastq.gz\n",
      "Total reads:            \t3182976\n",
      "Successfully aligned reads:\t2759808\t87%\t(>NC_000962.3 Mycobacterium tuberculosis H37Rv, complete genome)\n",
      "\tAligning (sense) to protein-coding genes:\t2%\n",
      "\tAligning (antisense) to protein-coding genes:\t15%\n",
      "\tAligning (sense) to ribosomal RNAs:\t0%\n",
      "\tAligning (antisense) to ribosomal RNAs:\t0%\n",
      "\tAligning (sense) to transfer RNAs:\t0%\n",
      "\tAligning (antisense) to transfer RNAs:\t0%\n",
      "\tAligning (sense) to miscellaneous RNAs:\t0%\n",
      "\tAligning (antisense) to miscellaneous RNAs:\t81%\n",
      "\tAligning to unannotated regions:        \t2%\n",
      "\n",
      "Aligning sequencing reads from files:\n",
      "\t8_S8_L001_R1_001.fastq.gz\n",
      "\t8_S8_L001_R2_001.fastq.gz\n",
      "Total reads:            \t2880959\n",
      "Successfully aligned reads:\t2443610\t85%\t(>NC_000962.3 Mycobacterium tuberculosis H37Rv, complete genome)\n",
      "\tAligning (sense) to protein-coding genes:\t1%\n",
      "\tAligning (antisense) to protein-coding genes:\t9%\n",
      "\tAligning (sense) to ribosomal RNAs:\t0%\n",
      "\tAligning (antisense) to ribosomal RNAs:\t0%\n",
      "\tAligning (sense) to transfer RNAs:\t0%\n",
      "\tAligning (antisense) to transfer RNAs:\t0%\n",
      "\tAligning (sense) to miscellaneous RNAs:\t0%\n",
      "\tAligning (antisense) to miscellaneous RNAs:\t89%\n",
      "\tAligning to unannotated regions:        \t1%\n",
      "\n",
      "Aligning sequencing reads from files:\n",
      "\t2_S2_L001_R1_001.fastq.gz\n",
      "\t2_S2_L001_R2_001.fastq.gz\n",
      "Total reads:            \t3124230\n",
      "Successfully aligned reads:\t1876505\t60%\t(>NC_000962.3 Mycobacterium tuberculosis H37Rv, complete genome)\n",
      "\tAligning (sense) to protein-coding genes:\t1%\n",
      "\tAligning (antisense) to protein-coding genes:\t5%\n",
      "\tAligning (sense) to ribosomal RNAs:\t0%\n",
      "\tAligning (antisense) to ribosomal RNAs:\t0%\n",
      "\tAligning (sense) to transfer RNAs:\t0%\n",
      "\tAligning (antisense) to transfer RNAs:\t0%\n",
      "\tAligning (sense) to miscellaneous RNAs:\t0%\n",
      "\tAligning (antisense) to miscellaneous RNAs:\t93%\n",
      "\tAligning to unannotated regions:        \t1%\n",
      "\n",
      "Analyzing transcripts...\n",
      "\n",
      "************************************************************\n",
      "\n",
      "************************************************************\n",
      "\n",
      "Computing transcript boundaries...\n",
      "\tNumber of 5'UTRs:\t\t\t4\n",
      "\tNumber of 3'UTRs:\t\t\t15\n",
      "\tNumber of predicted RNAs:\t\t\t268\n",
      "\tNumber of predicted RNAs (not antisense):\t\t20\n",
      "\tNumber of predicted RNAs (antisense):\t\t248\n",
      "\n",
      "Computing differential gene expression...\n",
      "\tNumber of differentially expressed protein coding genes:\t1345\n",
      "\n",
      "Computing likely operons...\n",
      "\tNumber of gene-pairs predicted to be part of the same operon:\t1575\n",
      "\tNumber of predicted multi-gene operons:\t\t885\n",
      "\n",
      "\n",
      "Summary of results written to file:\t/staging/vivek/RNA-Seq/MTB/mtuTemp/fastq/Results_cond0.1_6/summary.txt\n",
      "Expression of transcripts written to file:\t/staging/vivek/RNA-Seq/MTB/mtuTemp/fastq/Results_cond0.1_6/_transcripts.txt\n",
      "Operons written to file:\t\t/staging/vivek/RNA-Seq/MTB/mtuTemp/fastq/Results_cond0.1_6/_operons.txt\n",
      "\n",
      "FINISHED.\n",
      "\n"
     ]
    }
   ],
   "source": [
    "!java -Xmx150G -cp /staging/source/Rockhopper/Rockhopper.jar Rockhopper \\\n",
    "    -p 40 -g $ref -v TRUE -o $result1 -L Cond_0.1,Cond_6 \\\n",
    "    $fastq/4_S4_L001_R1_001.fastq.gz%$fastq/4_S4_L001_R2_001.fastq.gz,\\\n",
    "    $fastq/2_S2_L001_R1_001.fastq.gz%$fastq/2_S2_L001_R2_001.fastq.gz,\\\n",
    "    $fastq/1_S1_L001_R1_001.fastq.gz%$fastq/1_S1_L001_R2_001.fastq.gz \\\n",
    "    $fastq/8_S8_L001_R1_001.fastq.gz%$fastq/8_S8_L001_R2_001.fastq.gz,\\\n",
    "    $fastq/2_S2_L001_R1_001.fastq.gz%$fastq/2_S2_L001_R2_001.fastq.gz"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Cond_50_vs_cond_6"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Aligning sequencing reads from files:\n",
      "\t9_S9_L001_R1_001.fastq.gz\n",
      "\t9_S9_L001_R2_001.fastq.gz\n",
      "Total reads:            \t2396948\n",
      "Successfully aligned reads:\t1667804\t70%\t(>NC_000962.3 Mycobacterium tuberculosis H37Rv, complete genome)\n",
      "\tAligning (sense) to protein-coding genes:\t2%\n",
      "\tAligning (antisense) to protein-coding genes:\t19%\n",
      "\tAligning (sense) to ribosomal RNAs:\t0%\n",
      "\tAligning (antisense) to ribosomal RNAs:\t0%\n",
      "\tAligning (sense) to transfer RNAs:\t0%\n",
      "\tAligning (antisense) to transfer RNAs:\t0%\n",
      "\tAligning (sense) to miscellaneous RNAs:\t0%\n",
      "\tAligning (antisense) to miscellaneous RNAs:\t76%\n",
      "\tAligning to unannotated regions:        \t2%\n",
      "\n",
      "Aligning sequencing reads from files:\n",
      "\t6_S6_L001_R1_001.fastq.gz\n",
      "\t6_S6_L001_R2_001.fastq.gz\n",
      "Total reads:            \t2786683\n",
      "Successfully aligned reads:\t2447506\t88%\t(>NC_000962.3 Mycobacterium tuberculosis H37Rv, complete genome)\n",
      "\tAligning (sense) to protein-coding genes:\t2%\n",
      "\tAligning (antisense) to protein-coding genes:\t16%\n",
      "\tAligning (sense) to ribosomal RNAs:\t0%\n",
      "\tAligning (antisense) to ribosomal RNAs:\t0%\n",
      "\tAligning (sense) to transfer RNAs:\t0%\n",
      "\tAligning (antisense) to transfer RNAs:\t0%\n",
      "\tAligning (sense) to miscellaneous RNAs:\t0%\n",
      "\tAligning (antisense) to miscellaneous RNAs:\t80%\n",
      "\tAligning to unannotated regions:        \t2%\n",
      "\n",
      "Aligning sequencing reads from files:\n",
      "\t3_S3_L001_R1_001.fastq.gz\n",
      "\t3_S3_L001_R2_001.fastq.gz\n",
      "Total reads:            \t3135741\n",
      "Successfully aligned reads:\t2697854\t86%\t(>NC_000962.3 Mycobacterium tuberculosis H37Rv, complete genome)\n",
      "\tAligning (sense) to protein-coding genes:\t2%\n",
      "\tAligning (antisense) to protein-coding genes:\t14%\n",
      "\tAligning (sense) to ribosomal RNAs:\t0%\n",
      "\tAligning (antisense) to ribosomal RNAs:\t0%\n",
      "\tAligning (sense) to transfer RNAs:\t0%\n",
      "\tAligning (antisense) to transfer RNAs:\t0%\n",
      "\tAligning (sense) to miscellaneous RNAs:\t0%\n",
      "\tAligning (antisense) to miscellaneous RNAs:\t82%\n",
      "\tAligning to unannotated regions:        \t2%\n",
      "\n",
      "Aligning sequencing reads from files:\n",
      "\t8_S8_L001_R1_001.fastq.gz\n",
      "\t8_S8_L001_R2_001.fastq.gz\n",
      "Total reads:            \t2880959\n",
      "Successfully aligned reads:\t2443610\t85%\t(>NC_000962.3 Mycobacterium tuberculosis H37Rv, complete genome)\n",
      "\tAligning (sense) to protein-coding genes:\t1%\n",
      "\tAligning (antisense) to protein-coding genes:\t9%\n",
      "\tAligning (sense) to ribosomal RNAs:\t0%\n",
      "\tAligning (antisense) to ribosomal RNAs:\t0%\n",
      "\tAligning (sense) to transfer RNAs:\t0%\n",
      "\tAligning (antisense) to transfer RNAs:\t0%\n",
      "\tAligning (sense) to miscellaneous RNAs:\t0%\n",
      "\tAligning (antisense) to miscellaneous RNAs:\t89%\n",
      "\tAligning to unannotated regions:        \t1%\n",
      "\n",
      "Aligning sequencing reads from files:\n",
      "\t2_S2_L001_R1_001.fastq.gz\n",
      "\t2_S2_L001_R2_001.fastq.gz\n",
      "Total reads:            \t3124230\n",
      "Successfully aligned reads:\t1876505\t60%\t(>NC_000962.3 Mycobacterium tuberculosis H37Rv, complete genome)\n",
      "\tAligning (sense) to protein-coding genes:\t1%\n",
      "\tAligning (antisense) to protein-coding genes:\t5%\n",
      "\tAligning (sense) to ribosomal RNAs:\t0%\n",
      "\tAligning (antisense) to ribosomal RNAs:\t0%\n",
      "\tAligning (sense) to transfer RNAs:\t0%\n",
      "\tAligning (antisense) to transfer RNAs:\t0%\n",
      "\tAligning (sense) to miscellaneous RNAs:\t0%\n",
      "\tAligning (antisense) to miscellaneous RNAs:\t93%\n",
      "\tAligning to unannotated regions:        \t1%\n",
      "\n",
      "Analyzing transcripts...\n",
      "\n",
      "************************************************************\n",
      "\n",
      "************************************************************\n",
      "\n",
      "Computing transcript boundaries...\n",
      "\tNumber of 5'UTRs:\t\t\t4\n",
      "\tNumber of 3'UTRs:\t\t\t15\n",
      "\tNumber of predicted RNAs:\t\t\t400\n",
      "\tNumber of predicted RNAs (not antisense):\t\t23\n",
      "\tNumber of predicted RNAs (antisense):\t\t377\n",
      "\n",
      "Computing differential gene expression...\n",
      "\tNumber of differentially expressed protein coding genes:\t1278\n",
      "\n",
      "Computing likely operons...\n",
      "\tNumber of gene-pairs predicted to be part of the same operon:\t1566\n",
      "\tNumber of predicted multi-gene operons:\t\t892\n",
      "\n",
      "\n",
      "Summary of results written to file:\t/staging/vivek/RNA-Seq/MTB/mtuTemp/fastq/Results_cond50_6/summary.txt\n",
      "Expression of transcripts written to file:\t/staging/vivek/RNA-Seq/MTB/mtuTemp/fastq/Results_cond50_6/_transcripts.txt\n",
      "Operons written to file:\t\t/staging/vivek/RNA-Seq/MTB/mtuTemp/fastq/Results_cond50_6/_operons.txt\n",
      "\n",
      "FINISHED.\n",
      "\n"
     ]
    }
   ],
   "source": [
    "!java -Xmx150G -cp /staging/source/Rockhopper/Rockhopper.jar Rockhopper \\\n",
    "    -p 40 -g $ref -v TRUE -o $result2 -L Cond_50,Cond_6 \\\n",
    "    $fastq/9_S9_L001_R1_001.fastq.gz%$fastq/9_S9_L001_R2_001.fastq.gz,\\\n",
    "    $fastq/6_S6_L001_R1_001.fastq.gz%$fastq/6_S6_L001_R2_001.fastq.gz,\\\n",
    "    $fastq/3_S3_L001_R1_001.fastq.gz%$fastq/3_S3_L001_R2_001.fastq.gz \\\n",
    "    $fastq/8_S8_L001_R1_001.fastq.gz%$fastq/8_S8_L001_R2_001.fastq.gz,\\\n",
    "    $fastq/2_S2_L001_R1_001.fastq.gz%$fastq/2_S2_L001_R2_001.fastq.gz"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Cond_50_vs_Cond_0.1"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Aligning sequencing reads from files:\n",
      "\t9_S9_L001_R1_001.fastq.gz\n",
      "\t9_S9_L001_R2_001.fastq.gz\n",
      "Total reads:            \t2396948\n",
      "Successfully aligned reads:\t1667804\t70%\t(>NC_000962.3 Mycobacterium tuberculosis H37Rv, complete genome)\n",
      "\tAligning (sense) to protein-coding genes:\t2%\n",
      "\tAligning (antisense) to protein-coding genes:\t19%\n",
      "\tAligning (sense) to ribosomal RNAs:\t0%\n",
      "\tAligning (antisense) to ribosomal RNAs:\t0%\n",
      "\tAligning (sense) to transfer RNAs:\t0%\n",
      "\tAligning (antisense) to transfer RNAs:\t0%\n",
      "\tAligning (sense) to miscellaneous RNAs:\t0%\n",
      "\tAligning (antisense) to miscellaneous RNAs:\t76%\n",
      "\tAligning to unannotated regions:        \t2%\n",
      "\n",
      "Aligning sequencing reads from files:\n",
      "\t6_S6_L001_R1_001.fastq.gz\n",
      "\t6_S6_L001_R2_001.fastq.gz\n",
      "Total reads:            \t2786683\n",
      "Successfully aligned reads:\t2447506\t88%\t(>NC_000962.3 Mycobacterium tuberculosis H37Rv, complete genome)\n",
      "\tAligning (sense) to protein-coding genes:\t2%\n",
      "\tAligning (antisense) to protein-coding genes:\t16%\n",
      "\tAligning (sense) to ribosomal RNAs:\t0%\n",
      "\tAligning (antisense) to ribosomal RNAs:\t0%\n",
      "\tAligning (sense) to transfer RNAs:\t0%\n",
      "\tAligning (antisense) to transfer RNAs:\t0%\n",
      "\tAligning (sense) to miscellaneous RNAs:\t0%\n",
      "\tAligning (antisense) to miscellaneous RNAs:\t80%\n",
      "\tAligning to unannotated regions:        \t2%\n",
      "\n",
      "Aligning sequencing reads from files:\n",
      "\t3_S3_L001_R1_001.fastq.gz\n",
      "\t3_S3_L001_R2_001.fastq.gz\n",
      "Total reads:            \t3135741\n",
      "Successfully aligned reads:\t2697854\t86%\t(>NC_000962.3 Mycobacterium tuberculosis H37Rv, complete genome)\n",
      "\tAligning (sense) to protein-coding genes:\t2%\n",
      "\tAligning (antisense) to protein-coding genes:\t14%\n",
      "\tAligning (sense) to ribosomal RNAs:\t0%\n",
      "\tAligning (antisense) to ribosomal RNAs:\t0%\n",
      "\tAligning (sense) to transfer RNAs:\t0%\n",
      "\tAligning (antisense) to transfer RNAs:\t0%\n",
      "\tAligning (sense) to miscellaneous RNAs:\t0%\n",
      "\tAligning (antisense) to miscellaneous RNAs:\t82%\n",
      "\tAligning to unannotated regions:        \t2%\n",
      "\n",
      "Aligning sequencing reads from files:\n",
      "\t4_S4_L001_R1_001.fastq.gz\n",
      "\t4_S4_L001_R2_001.fastq.gz\n",
      "Total reads:            \t3061062\n",
      "Successfully aligned reads:\t2667646\t87%\t(>NC_000962.3 Mycobacterium tuberculosis H37Rv, complete genome)\n",
      "\tAligning (sense) to protein-coding genes:\t1%\n",
      "\tAligning (antisense) to protein-coding genes:\t12%\n",
      "\tAligning (sense) to ribosomal RNAs:\t0%\n",
      "\tAligning (antisense) to ribosomal RNAs:\t0%\n",
      "\tAligning (sense) to transfer RNAs:\t0%\n",
      "\tAligning (antisense) to transfer RNAs:\t0%\n",
      "\tAligning (sense) to miscellaneous RNAs:\t0%\n",
      "\tAligning (antisense) to miscellaneous RNAs:\t84%\n",
      "\tAligning to unannotated regions:        \t2%\n",
      "\n",
      "Aligning sequencing reads from files:\n",
      "\t2_S2_L001_R1_001.fastq.gz\n",
      "\t2_S2_L001_R2_001.fastq.gz\n",
      "Total reads:            \t3124230\n",
      "Successfully aligned reads:\t1876505\t60%\t(>NC_000962.3 Mycobacterium tuberculosis H37Rv, complete genome)\n",
      "\tAligning (sense) to protein-coding genes:\t1%\n",
      "\tAligning (antisense) to protein-coding genes:\t5%\n",
      "\tAligning (sense) to ribosomal RNAs:\t0%\n",
      "\tAligning (antisense) to ribosomal RNAs:\t0%\n",
      "\tAligning (sense) to transfer RNAs:\t0%\n",
      "\tAligning (antisense) to transfer RNAs:\t0%\n",
      "\tAligning (sense) to miscellaneous RNAs:\t0%\n",
      "\tAligning (antisense) to miscellaneous RNAs:\t93%\n",
      "\tAligning to unannotated regions:        \t1%\n",
      "\n",
      "Aligning sequencing reads from files:\n",
      "\t1_S1_L001_R1_001.fastq.gz\n",
      "\t1_S1_L001_R2_001.fastq.gz\n",
      "Total reads:            \t3182976\n",
      "Successfully aligned reads:\t2759808\t87%\t(>NC_000962.3 Mycobacterium tuberculosis H37Rv, complete genome)\n",
      "\tAligning (sense) to protein-coding genes:\t2%\n",
      "\tAligning (antisense) to protein-coding genes:\t15%\n",
      "\tAligning (sense) to ribosomal RNAs:\t0%\n",
      "\tAligning (antisense) to ribosomal RNAs:\t0%\n",
      "\tAligning (sense) to transfer RNAs:\t0%\n",
      "\tAligning (antisense) to transfer RNAs:\t0%\n",
      "\tAligning (sense) to miscellaneous RNAs:\t0%\n",
      "\tAligning (antisense) to miscellaneous RNAs:\t81%\n",
      "\tAligning to unannotated regions:        \t2%\n",
      "\n",
      "Analyzing transcripts...\n",
      "\n",
      "************************************************************\n",
      "\n",
      "************************************************************\n",
      "\n",
      "Computing transcript boundaries...\n",
      "\tNumber of 5'UTRs:\t\t\t51\n",
      "\tNumber of 3'UTRs:\t\t\t140\n",
      "\tNumber of predicted RNAs:\t\t\t317\n",
      "\tNumber of predicted RNAs (not antisense):\t\t13\n",
      "\tNumber of predicted RNAs (antisense):\t\t304\n",
      "\n",
      "Computing differential gene expression...\n",
      "\tNumber of differentially expressed protein coding genes:\t1532\n",
      "\n",
      "Computing likely operons...\n",
      "\tNumber of gene-pairs predicted to be part of the same operon:\t1555\n",
      "\tNumber of predicted multi-gene operons:\t\t881\n",
      "\n",
      "\n",
      "Summary of results written to file:\t/staging/vivek/RNA-Seq/MTB/mtuTemp/fastq/Results_cond50_0.1/summary.txt\n",
      "Expression of transcripts written to file:\t/staging/vivek/RNA-Seq/MTB/mtuTemp/fastq/Results_cond50_0.1/_transcripts.txt\n",
      "Operons written to file:\t\t/staging/vivek/RNA-Seq/MTB/mtuTemp/fastq/Results_cond50_0.1/_operons.txt\n",
      "\n",
      "FINISHED.\n",
      "\n"
     ]
    }
   ],
   "source": [
    "!java -Xmx150G -cp /staging/source/Rockhopper/Rockhopper.jar Rockhopper \\\n",
    "    -p 40 -g $ref -v TRUE -o $result3 -L Cond_50,Cond_0.1 \\\n",
    "    $fastq/9_S9_L001_R1_001.fastq.gz%$fastq/9_S9_L001_R2_001.fastq.gz,\\\n",
    "    $fastq/6_S6_L001_R1_001.fastq.gz%$fastq/6_S6_L001_R2_001.fastq.gz,\\\n",
    "    $fastq/3_S3_L001_R1_001.fastq.gz%$fastq/3_S3_L001_R2_001.fastq.gz \\\n",
    "    $fastq/4_S4_L001_R1_001.fastq.gz%$fastq/4_S4_L001_R2_001.fastq.gz,\\\n",
    "    $fastq/2_S2_L001_R1_001.fastq.gz%$fastq/2_S2_L001_R2_001.fastq.gz,\\\n",
    "    $fastq/1_S1_L001_R1_001.fastq.gz%$fastq/1_S1_L001_R2_001.fastq.gz"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.1"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
